src/Makefile.in

```diff
@@ -218,6 +220,7 @@ opencl_%.h:	| ../run/kernels
 
 ../run/kernels:
 	$(MKDIR) ../run/kernels
+	$(CP) jumbo/opencl/include/opencl_*.h ../run/kernels  #TODO: hack to make it work
```

Comment:
Why is it needed? The copy above works for `.cl` files but not for `.h` files. Failed to see why!